### PR TITLE
Compute circle icon to reuse symbol layer

### PR DESCRIPTION
**Pull request by AliceR**
This way we don't need to filter, plus with a fallback symbol we are more resilient to api naming changes (we might display the wrong symbol, but it's better than nothing, isn't it?).
Reducing the number of layers could positively impact the performance as well.

Plus: we could extend this more easily* for new symbols, if we ever were to introduce triangles or even stars 😬 
(* curious how complex the algorithm for a star would be)

What I did not touch is the nearby locations map, as @necoline is working on that one. There, we currently get only black squares if the parameter isn't available.

👉 I noticed that currently we don't seem to have values in the last two days. To test this with color scheme set [this value](https://github.com/openaq/openaq.org/blob/develop/app/assets/scripts/utils/colors.js#L106) to `months` instead of days

### CODE REVIEW

File: app/assets/scripts/components/map/measurements-layer.js [hunk 1]
```diff
@@ -23,115 +16,75 @@ export default function MeasurementsLayer({
...
-  const squaresFilter = ['==', ['get', 'sensorType'], 'low-cost sensor'];
-
-  const circlesCountryFilter = ['all', countryFilter, circlesFilter];
-  const squaresCountryFilter = ['all', countryFilter, squaresFilter];
+  const iconMatch = [
+    'match',
+    ['get', 'sensorType'],
+    'low-cost sensor',
+    'square',
+    'reference grade',
```
Comment (AliceR) [inline app/assets/scripts/components/map/measurements-layer.js:24]:
```suggestion
    'reference',
```
The backend settled on `reference`.

**Review by rwegener2 — APPROVED** [review]
To this seems to look the same (maybe a bit faster 🤷‍♀️ ), I believe that means that I approve!

File: app/assets/scripts/components/map/measurements-layer.js [hunk 2]
```diff
@@ -23,115 +16,75 @@ export default function MeasurementsLayer({
...
-  const squaresFilter = ['==', ['get', 'sensorType'], 'low-cost sensor'];
-
-  const circlesCountryFilter = ['all', countryFilter, circlesFilter];
-  const squaresCountryFilter = ['all', countryFilter, squaresFilter];
+  const iconMatch = [
+    'match',
+    ['get', 'sensorType'],
+    'low-cost sensor',
+    'square',
+    'reference',
```
[the remark this answers is not in the document]
Comment (necoline) [reply app/assets/scripts/components/map/measurements-layer.js:24]: There was a conversation on this on Friday and there was an obscure update that Bitner needed to make to ensure the sensor type was reference grade across the board

[the remark this answers is not in the document]
Comment (AliceR) [reply app/assets/scripts/components/map/measurements-layer.js]: Ok, reverted with 2a64758

**Pull request merged**